test/integration/controller/job/job_controller_test.go

```diff
@@ -221,3 +244,38 @@ var _ = ginkgo.Describe("Job controller", func() {
 		}, framework.Timeout, framework.Interval).Should(gomega.BeTrue())
 	})
 })
+
+var _ = ginkgo.Describe("Job controller for workloads with no queue set", func() {
```

Comment:
nit: I would use a single `ginkgo.Describe` with two `ginkgo.Context`, because you are describing the same controller.

Reply:
the indentation is too much for me :)